src/data/Dhis2TrackedEntityInstances.ts

```diff
@@ -589,7 +589,7 @@ function getValue(
     dataValue: { optionId?: string; value: EventDataValue["value"] },
     optionById: Record<Id, { id: Id; code: string } | undefined>
 ): string {
-    if (dataValue.optionId && dataValue.optionId !== "true" && dataValue.optionId !== "false") {
+    if (dataValue.optionId) {
```

Comment:
The comment in the PR says "Parse yes/no values to true/false to avoid validation errors with boolean attributes", but I don't see how just removing this check fixes it (why was the code there, then?)